docs/toctree.rst

```diff
@@ -0,0 +1,48 @@
+=====
+Vyper
+=====
+
+.. toctree::
+    :maxdepth: 2
+
+    Overview <index.rst>
+
+.. toctree::
+    :caption: Getting Started
+    :maxdepth: 2
+
+    installing-vyper.rst
+    vyper-by-example.rst
+
+.. toctree::
+    :caption: Language Description
+    :maxdepth: 2
+
+    structure-of-a-contract.rst
+    types.rst
+    constants-and-vars.rst
+    statements.rst
+    control-structures.rst
```

Comment:
Should these two be switched?

Reply:
Python and Solidity docs both start small and get bigger, I followed their lead.  I think it makes sense..  you need the small pieces to understand the larger ones.